middleware.js

```diff
@@ -53,6 +54,7 @@ function asyncMiddleware (fn) {
 
 function logErrors (err, req, res, next) {
   log.error("error", {stack: err.stack});
+  Sentry.captureException(err);
```

Comment:
Do we want to capture stacks on dev/stage/prod? or just stage/prod? prod only?
Or are we just overriding the `SENTRY_DSN` env var on production so it routes to a different bucket?

Reply:
I'd like to, but I can't tell from [the docs](https://docs.sentry.io/learn/capturing/?platform=node) how we're supposed to pass the stacktraces into `captureException` 😢